app/helpers/wizard/sessions_speakers.py

```diff
@@ -111,9 +113,3 @@ def save_custom_forms(data, event_id):
     custom_forms.speaker_form = json.dumps(data['speaker'])
     save_to_db(custom_forms)
 
-
-def delete_all_sessions_speakers_data(event_id):
-    Microlocation.query.filter_by(event_id=event_id).delete()
-    SessionType.query.filter_by(event_id=event_id).delete()
-    Track.query.filter_by(event_id=event_id).delete()
-    CallForPaper.query.filter_by(event_id=event_id).delete()
```

Comment:
This function was sitting randomly, without being used anywhere, that's why its been deleted.